example_plugin.py

```diff
@@ -3,16 +3,28 @@
 import sys
 import typing
 from dataclasses import dataclass
-from arcaflow_plugin_sdk import plugin, validation, annotations
+from arcaflow_plugin_sdk import plugin, validation, annotations, schema
 
 
 @dataclass
 class FullName:
     """
     A full name holds the first and last name of an individual.
     """
-    first_name: typing.Annotated[str, validation.min(1), validation.pattern(re.compile("[a-zA-Z]"))]
-    last_name: typing.Annotated[str, validation.min(1), validation.pattern(re.compile("[a-zA-Z]"))]
+    first_name: typing.Annotated[
```

Comment:
This could have been splitted in 2 PRs: one for changing the formatting style and one for adding the 2 schema entries. It would have made more evident what was changing here.